mm2src/mm2_main/src/lp_swap.rs

```diff
@@ -1221,8 +1232,9 @@ pub async fn latest_swaps_for_pair(
         Err(_) => return Err(MmError::new(LatestSwapsErr::UnableToQuerySwapStorage)),
     };
 
-    let mut swaps = Vec::with_capacity(db_result.uuids.len());
-    for uuid in db_result.uuids.iter() {
+    let mut swaps = Vec::with_capacity(db_result.uuids_and_types.len());
+    // TODO this is needed for trading bot, which seems not used as of now. Remove the code?
```

Comment:
I think we should the keep the code as the plan for the trading bot and whether we will enhance the current implementation or implement a different one is not clear yet. If we want to optimized builds without this unnecessary code we can add `trading-bot` feature flag that is disabled by default.

Reply:
Ok, I will keep the current code because it is out of the current task scope.